opus/application/apps/ui/views.py

```diff
@@ -1079,9 +1079,12 @@ def _escape_or_label_results(old_slug, pi):
     if final_msg == '':
         final_msg = None
 
+    reqno = get_reqno(request)
+
```

Comment:
Generally we've been either requiring reqno to be present, or requiring it to be absent, so we don't have cases where we accidentally leave it off and then it causes a bug. The general code looks like this:
```
    if reqno is None:
        log.error('api_cart_status: Missing or badly formatted reqno')
        ret = Http404(settings.HTTP404_MISSING_REQNO)
        exit_api_call(api_code, ret)
        raise ret
```
But if we put that code here, we also need to update all the normalizeurl tests in test_ui_api.py to include a reqno. So we should really decide now, if we can, if the reqno is needed.